.nvmrc

```diff
@@ -1 +1 @@
-10.17
+12.20.1
```

Comment:
@jrylander Why did you change the default version for local development? Do you have any limitations with 10? We still supported Node.js 8+...

Reply:
I looked at what is used in dev.Dockerfile and thought synching those would be a good idea. I see your point, should dev.Dockerfile still use 12?

Reply:
Oh, ok. Forget about that. let's use 12 then.